primitives/src/currency.rs

```diff
@@ -205,6 +205,8 @@ create_currency_id! {
 		PHA("Phala Native Token", 12) = 170,
 		KINT("Kintsugi Native Token", 12) = 171,
 		KBTC("Kintsugi Wrapped BTC", 8) = 172,
+		RMRK("Rmrk", 10) = 173,
```

Comment:
why add those?

Reply:
deleted now. when I test statemine prefix location break change. I use RMRK and RMRK_V2 as location migration.